ui/src/components/ripple-out/ProgressSection.js

```diff
@@ -325,6 +293,34 @@ const ProgressSection = ({ bodyFontSize }) => {
                     </Box>
                   </Box>
                 </Flex>
+                <Box
+                  textAlign="center"
+                  mt={[10, 20]}
+                >
+                  <Flex width="100%" justifyContent="center">
+                    <Text fontSize={['xl', '3xl']} fontWeight="thin" width={["100%", "50%"]}>
+                      Click below to learn more how to participate in the Ripple Out Campaign!
+                    </Text>
+                  </Flex>
+                  <Box textAlign={['center', 'center']} width="100%" mt={5}>
+                    <Button
+                      mt={5}
+                      mb={10}
+                      href="https://bit.ly/ripple-pledge"
```

Comment:
this doesn't work btw. you need to use the `onClick` handler and do `router.push` i think https://github.com/chakra-ui/chakra-ui/discussions/3292 or put a link component in the button

Reply:
resolved.